interfaces/builtin/fwupd.go

```diff
@@ -76,6 +76,7 @@ const fwupdPermanentSlotAppArmor = `
   /boot/efi/EFI/boot/fwupd*.efi* rw,
   /boot/efi/EFI/boot/fw/** rw,
   # allow access to fwupd* and fw/ under ubuntu/ for classic systems
+  # but it should be deprecated once old uefi-fw-tools is no longer used
   /boot/efi/EFI/ubuntu/fwupd*.efi* rw,
   /boot/efi/EFI/ubuntu/fw/** rw,
```

Comment:
I would think that these paths should actually go away since they're implied by the `fwupdPermanentSlotAppArmorClassic` no?

Reply:
Unfortunately some Dell Edge gateway were still using latest track which was based on 0.7.x version, I'd like to remove them if Dell agrees that they can be switched `18` track on the field one day.